moveit_ros/planning/planning_scene_monitor/test/current_state_monitor_test.cpp

```diff
@@ -146,10 +146,17 @@ TEST_F(CurrentStateMonitorTest, StateUpdateTest)
 TEST_F(CurrentStateMonitorTest, IncrementalTimeStamps)
 {
   sendJointStateAndWait(js_a);
-  EXPECT_EQ(js_a.header.stamp, csm->getCurrentStateTime());
+  EXPECT_EQ(js_a.header.stamp, csm->getCurrentStateTime())
+      << "older partial joint state was ignored in current state retrieval!";
```

Comment:
I think this comment doesn't fit (anymore?).

Reply:
Yup

Reply:
Removed.